### ISSUE TITLE
details access system

### ISSUE THREAD

**Issue by m3morx** [mailwatch/1.2.0#1094]
Is it possible to implement a more details access right system ?
I.e. to grant a user just to view the quarantine to release a message if necessary?

**Comment by Skywalker-11** [comment]
It should already be possible for a user to release mails that are addressed to him a are quarantined. 

Or do you mean it more like a domain admin who can't edit black/white list and users?

**Comment by m3morx** [comment]
Correct. A domain admin who can have various roles like, edit generate Reports or release quarantiend emails for the whole Domain. But do not have access to the entire Maillog i.e.
Thats what i've lokking for.

**Comment by Skywalker-11** [comment]
It may be included in the 2.0 release

**Comment by endelwar** [comment]
In mailwatch 2.0 we can think about an acl system, where User roles are collections of permission, and any single installation can have its own custom roles

**Comment by m3morx** [comment]
great. looking forward to have this implemented. ty
